cmd/snap/cmd_info.go

```diff
@@ -417,6 +417,22 @@ func (iw *infoWriter) printDescr() {
 	printDescr(iw, iw.theSnap.Description, iw.termWidth)
 }
 
+func (iw *infoWriter) maybePrintCategories() {
```

Comment:
Is this information important enough to show by default? Would it make sense to gate this behind the `--verbose` flag (as exposed through `iw.verbose`)?

It'd also be good to include a simple test for this function.

Reply:
I don't know - for my purposes I don't need to see this, but it felt like it might be useful. Happy to put it behind verbose or drop it.